.github/workflows/ci.yml

```
@@ -92,3 +92,44 @@ jobs:
           DEFAULT_BRANCH: master
           # Not needed here as only one Linter is used.
           #GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  run-examples:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get update;
+          DEBIAN_FRONTEND=noninteractive sudo apt-get install --no-install-recommends -yq \
+            jq \
+            libasound2-dev \
+            libudev-dev \
+            wget \
+            unzip \
+            xvfb;
```

```suggestion
            jq libasound2-dev libudev-dev wget unzip xvfb;
```
takes a bit less space